docs-v2/docusaurus.config.js

```diff
@@ -30,7 +30,7 @@ const config = {
     'Apache Superset is a modern data exploration and visualization platform',
   url: 'https://superset.apache.org',
   baseUrl: '/',
-  onBrokenLinks: 'throw',
```

Comment:
should we change this? throwing if we build docs with broken links seems reasonable to me

Reply:
We can change it back when all the other pending PRs are merged as some links have been removed there. I'll take care of that